archinstall/lib/menu/selection_menu.py

```diff
@@ -544,31 +582,37 @@ def exit_callback(self):
 					storage['arguments']['disk_layouts'], storage['arguments']['!encryption-password'])
 
 	def _install_text(self):
-		missing = self._missing_configs()
+		missing = len(self._missing_configs())
 		if missing > 0:
 			return _('Install ({} config(s) missing)').format(missing)
 		return 'Install'
 
-	def _missing_configs(self):
+	def _prev_install_missing_config(self) -> Optional[str]:
+		if missing := self._missing_configs():
+			text = str(_('Missing configurations:\n'))
+			for m in missing:
+				text += f'- {m}\n'
+			return text[:-1]  # remove last new line
+		return None
+
+	def _missing_configs(self) -> List[str]:
 		def check(s):
 			return self._menu_options.get(s).has_selection()
 
-		missing = 0
+		missing = []
 		if not check('bootloader'):
-			missing += 1
+			missing += ['Bootloader']
 		if not check('hostname'):
-			missing += 1
+			missing += ['Hostname']
 		if not check('audio'):
-			missing += 1
-		if not check('timezone'):
-			missing += 1
+			missing += ['Audio']
 		if not check('!root-password') and not check('!superusers'):
-			missing += 1
+			missing += [str(_('Either root-password or at least 1 superuser must be specified'))]
 		if not check('harddrives'):
-			missing += 1
+			missing += ['Harddrives']
```

Comment:
Nit pick: This shouldn't be one word.

Reply:
Or, alternatively it could just be 'Drives'.

Reply:
Might need to be changed elsewhere too if we do. Visually it probably doesn't matter if we only change here. But the heck for `if .."harddrives"` has to be changed to `if .."hard drives"` as well where we rely on this keyword :)